CHANGELOG.md

```diff
@@ -89,7 +89,6 @@
 * [ENHANCEMENT] Querier: Cache the converted float histogram from chunk iterator, hence there is no need to lookup chunk every time to get the converted float histogram. #4684
 * [ENHANCEMENT] Ruler: Improve rule upload performance when not enforcing per-tenant rule group limits. #4828
 * [ENHANCEMENT] Improved memory limit on the in-memory cache used for regular expression matchers. #4751
-* [ENHANCEMENT] Go: update to 1.20.3. #4773
```

Comment:
Note to reviewer: I removed this because this changelog is overridden by changelog in line 104.